crafters/nlp/PDFExtractorSegmenter/tests/test_pdfextractorcrafter.py

```diff
@@ -91,6 +97,7 @@ def test_io_buffer_text():
 
     # Check test
     assert chunks[0]['text'] == expected_text
+    assert chunks[0]['mime_type'] == "text/plain"
```

Comment:
try to use single quote as per jina style

Reply:
Got it